README.md

```diff
@@ -54,7 +54,7 @@ All commands by default will output to standard out. You can pass the `--output
 
 Generates a country's trust anchor by supplying the location of its metadata and its certificate or multiple certificates if the country has a certificate chain.
 
-    ./verify-trust-anchor trust-anchor import "https://metadata.example.com/example-country.xml" path/to/signing.crt [path/to/signing_ca.crt [...]]
+    ./build/install/verify-eidas-trust-anchor/bin/verify-eidas-trust-anchor trust-anchor import --output=example.json "https://metadata.example.com/example-country.xml" path/to/signing.crt [path/to/signing_ca.crt [...]]
```

Comment:
The relative path is also the same in all other command examples – can we change the others too?

Reply:
Also, it says on line 51 about how to use output, do we need to put it in here explicitly?